docs/update_readme.py

```diff
@@ -0,0 +1,20 @@
+# Script to update end-to-end example in README.md.
```

Comment:
I am a bit unclear how this is supposed to work. I assume this is intended to be run by a person creating a PR, and the actual file in the repo should be the *updated* file?

Reply:
This makes sense, I add a separate workflow to check diffs and instruct the user to run update_readme if the ci fails